opensquirrel/decomposer/aba_decomposer.py

```diff
@@ -45,8 +58,10 @@ def get_decomposition_angles(self, alpha: float, axis: AxisLike) -> tuple[float,
 
         """
         axis = Axis(axis)
```

Comment:
Do we need this line? Can't we do `axis[index]` on an `AxisLike`?

Reply:
This makes an axis object from an `AxisLike` object. If it already is an `Axis`, then it does nothing, if it is not, it will try to cast it to an `Axis` and then we are sure that we can do stuff like `axis[index]`.

This makes the data predictable, we know for example dat the axis values are normalized and are floating points values.